static/src/javascripts/projects/commercial/modules/cmp/cmp.js

```diff
@@ -16,6 +16,20 @@ import { encodeVendorConsentData } from 'commercial/modules/cmp/cookie';
 
 import type { CmpConfig } from 'commercial/modules/cmp/types';
 
+type receivedMessage = {
+    data: {
+        __cmpCall: ?{
+            callId: any,
+            command: any,
+            parameter: any,
+        },
+    },
+    origin: string,
+    source: ?{
+        postMessage: (__cmpReturn: any, origin: string) => void,
+    },
+};
+
```

Comment:
I would normally try to avoid using the `any` type however this is the interface of our IAB CMP implementation, and various external invokers.

Ideally there will never be missing keys from the data object...  However the framework / API guidelines are fairly new, JavaScript is JavaScript and most implementations rely on the happy path.